packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -165,7 +165,9 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
 
     private async summarizeInternal(fullTree: boolean, trackState: boolean): Promise<ISummarizeInternalResult> {
         const summarizeResult = await this.summarizeFn(fullTree, trackState);
-        this.gcData = cloneGCData(summarizeResult.gcData);
+        if (summarizeResult.gcData !== undefined) {
```

Comment:
This is so that when we remove gcData from summarize, it becomes a non-breaking change and we don't have to wait another couple of versions to remove it.